.gitattributes

```diff
@@ -1,5 +1 @@
 * text=auto eol=lf
-/*/**/Dockerfile            linguist-generated
-/*/**/docker-entrypoint.sh  linguist-generated
-/Dockerfile*.template        linguist-language=Dockerfile
```

Comment:
Maybe we can leave this lone alone?

Reply:
yeah, that one should be kept

Reply:
Why? Template files aren't generated, and if someone is changing them, we need to look at the change